client/src/splitViewHandle.js

```diff
@@ -0,0 +1,169 @@
+import { LuigiClientBase } from './baseClass';
+import { helpers } from './helpers';
+
+/**
+ * The Split View 
+  - Actions to control the split view
+  - Event handlers
+  - Get the current state
+  * @name splitView
+  */
+export class splitViewHandle extends LuigiClientBase {
+  /**
+   * @private
+   */
+  constructor(settings) {
+    super();
+
+    this.validSplitViewEvents = ['expand', 'collapse', 'resize', 'close'];
+
+    this.splitView = {
+      exists: true,
+      size: 40,
+      collapsed: false
+    };
+
+    Object.assign(this, settings);
+
+    this.splitView.listeners = [
+      helpers.addEventListener(`luigi.navigation.splitview.internal`, e => {
+        // console.log('received new internal data', e.data.data);
+        Object.assign(this.splitView, e.data.data);
+      }),
+      helpers.addEventListener(`luigi.navigation.splitview.resize`, e => {
```

Comment:
Correct me if I am wrong, but I think we don't need the listeners for collapse and expand, since the fields are already updated via the `luigi.navigation.splitview.internal` handler.

Reply:
Also for the listeners we keep (except `internal`), we need to add the '.ok' suffix to the message name